src/callbacks_step/analysis.jl

```diff
@@ -414,6 +414,26 @@ function (cb::DiscreteCallback{Condition,Affect!})(sol) where {Condition, Affect
 end
 
 
+function calc_jacobian_volume(element, mesh::TreeMesh, equations, cache)
+  return inv(cache.elements.inverse_jacobian[element])^ndims(equations)
+end
+
+
+function calc_jacobian_volume(element, mesh::StructuredMesh, equations, cache)
+  return inv(cache.elements.inverse_jacobian[element])
+end
```

Comment:
I would move those functions to the `dg_structured` part, since they are not really related to the `AnalysisCallback` (they are just used here). Also, why did you include `equations`? Can't you just do `ndims(mesh)`?

Reply:
Why not just `jacobian_volume`?

Reply:
Well, I didn't want to appear too nitpicky and question all their naming choices, but since you are doing it anyways: Yes, `jacobian_volume` would IMHO be a better name ;-)

Reply:
Actually, maybe `volume_jacobian` would be even better, since it does not risk of being confused with "volume of the Jacobian"